### ISSUE TITLE
Broken build for Java 8 target

### ISSUE THREAD

**Issue by PoslavskySV** [Kotlin/dataframe#119]
Hi, 

version `0.8.0-rc-9` breaks builds for Java 8 targets:
```
➜  git:(master) ✗ ./gradlew clean build
> Task :kspKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':kspKotlin'.
> Error while evaluating property 'filteredArgumentsMap' of task ':kspKotlin'
   > Could not resolve all files for configuration ':compileClasspath'.
      > Could not resolve org.jetbrains.kotlinx:dataframe:0.8.0-rc-9.
        Required by:
            project :
         > No matching variant of org.jetbrains.kotlinx:dataframe:0.8.0-rc-9 was found. The consumer was configured to find an API of a library compatible with Java 8, preferably in the form of class files, preferably optimized for standard JVMs, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm' but:
             - Variant 'apiElements' capability org.jetbrains.kotlinx:dataframe:0.8.0-rc-9 declares an API of a library, packaged as a jar, preferably optimized for standard JVMs, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm':
                 - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 8
             - Variant 'javadocElements' capability org.jetbrains.kotlinx:dataframe:0.8.0-rc-9 declares a runtime of a component, and its dependencies declared externally:
                 - Incompatible because this component declares documentation and the consumer needed a library
                 - Other compatible attributes:
                     - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
                     - Doesn't say anything about its target Java version (required compatibility with Java 8)
                     - Doesn't say anything about its elements (required them preferably in the form of class files)
                     - Doesn't say anything about org.jetbrains.kotlin.platform.type (required 'jvm')
             - Variant 'runtimeElements' capability org.jetbrains.kotlinx:dataframe:0.8.0-rc-9 declares a runtime of a library, packaged as a jar, preferably optimized for standard JVMs, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm':
                 - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 8
             - Variant 'sourcesElements' capability org.jetbrains.kotlinx:dataframe:0.8.0-rc-9 declares a runtime of a component, and its dependencies declared externally:
                 - Incompatible because this component declares documentation and the consumer needed a library
                 - Other compatible attributes:
                     - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
                     - Doesn't say anything about its target Java version (required compatibility with Java 8)
                     - Doesn't say anything about its elements (required them preferably in the form of class files)
                     - Doesn't say anything about org.jetbrains.kotlin.platform.type (required 'jvm')
```

Is it possible to fix? Still many production envs use Java 8 and have no possibility to upgrade for another JVM version.

**Comment by koperagen** [comment]
Hi! It's not intentional and should be fixed in recent dev `0.8.0-dev-1037`. Can you try and tell me if it works?

**Comment by PoslavskySV** [comment]
Super - I confirm that with dev build it works as expected - thank you!
